src/coreclr/tools/aot/crossgen2/Program.cs

```diff
@@ -472,7 +472,10 @@ private int Run(string[] args)
                 {
                     try
                     {
-                        EcmaModule module = _typeSystemContext.GetModuleFromPath(referenceFile);
+                        EcmaModule module = _typeSystemContext.GetModuleFromPath(referenceFile, throwOnFailureToLoad: false);
```

Comment:
Do we still need this to be in a try/catch (`catch { } // Ignore non-managed pe files`)? Same question for below.

Reply:
I think that we still need a try/catch block. I'm really trying to reduce the number of weirdo useless exceptions, not get rid of them entirely. I'm not prepared at the moment to actually attempt to get rid of all of the exception paths that are possible (or at least make sure they are only present for cases where we'd like to kill the entire debugging process.